java/src/main/java/com/alibaba/graphar/readers/arrowchunk/VertexPropertyArrowChunkReader.java

```diff
@@ -0,0 +1,130 @@
+
+    Factory factory = FFITypeFactory.getFactory(VertexPropertyArrowChunkReader.class);
+
+    /**
+     * Sets chunk position indicator for reader by internal vertex id. If internal vertex id is not
+     * found, will return Status::IndexError error. After seeking to an invalid vertex id, the next
+     * call to GetChunk function may undefined, e.g. return a non exist path.
+     *
+     * @param id the vertex id.
+     */
+    @CXXValue
+    Status seek(@FFITypeAlias(GAR_ID_TYPE) long id);
+
+    /** Return the current arrow chunk table of chunk position indicator. */
+    @FFINameAlias("GetChunk")
+    @CXXValue
+    Result<StdSharedPtr<ArrowTable>> getChunk();
+
+    /**
+     * Get the vertex id range of current chunk.
+     *
+     * @return Result: std::pair<begin_id, end_id> or error.
+     */
+    @FFINameAlias("GetRange")
+    @CXXValue
+    @FFITypeAlias("GraphArchive::Result<std::pair<GraphArchive::IdType,GraphArchive::IdType>>")
+    Result<StdPair<Long, Long>> getRange();
+
+    /**
+     * Sets chunk position indicator to next chunk. if current chunk is the last chunk, will return
+     * Status::IndexError error.
+     */
+    @FFINameAlias("next_chunk")
+    @CXXValue
+    Status nextChunk();
+
+    /** Get the chunk number of current vertex property group. */
+    @FFINameAlias("GetChunkNum")
+    @FFITypeAlias(GAR_ID_TYPE)
+    long getChunkNum();
+
+    /**
+     * Helper function to Construct VertexPropertyArrowChunkReader.
+     *
+     * @param graphInfo The graph info to describe the graph.
+     * @param label label of the vertex.
+     * @param propertyGroup The property group of the vertex.
+     */
+    static Result<VertexPropertyArrowChunkReader> constructVertexPropertyArrowChunkReader(
+            @CXXReference GraphInfo graphInfo,
+            @CXXReference StdString label,
+            @CXXReference PropertyGroup propertyGroup) {
+        return GrapharStaticFunctions.INSTANCE.constructVertexPropertyArrowChunkReader(
+                graphInfo, label, propertyGroup);
+    }
+
+    @FFIFactory
+    interface Factory {
+        /**
+         * Initialize the VertexPropertyArrowChunkReader.
+         *
+         * @param vertexInfo The vertex info that describes the vertex type.
+         * @param propertyGroup The property group that describes the property group.
+         * @param prefix The absolute prefix.
+         */
+        VertexPropertyArrowChunkReader create(
+                @CXXReference VertexInfo vertexInfo,
+                @CXXReference PropertyGroup propertyGroup,
+                @CXXReference StdString prefix);
+
+        /**
+         * Initialize the VertexPropertyArrowChunkReader.
+         *
+         * @param vertexInfo The vertex info that describes the vertex type.
+         * @param propertyGroup The property group that describes the property group.
+         * @param prefix The absolute prefix.
+         */
+        //    VertexPropertyArrowChunkReader create(
```

Comment:
Can you add some comment here if the code has to remain?

Reply:
Removed